qiskit/optimization/problems/quadratic_program.py

```diff
@@ -794,7 +794,10 @@ def read_from_lp_file(self, filename: str) -> None:
             filename: The filename of the file to be loaded.
 
         Raises:
```

Comment:
It's good to add `RuntimeError` due to missing of cplex.
The traceback is as follows.
```
Traceback (most recent call last):
  File "l.py", line 4, in <module>
    mod.read_from_lp_file('test.lp')
  File "/home/ima/github/qiskit/aqua/qiskit/optimization/problems/quadratic_program.py", line 815, in read_from_lp_file
    model = model_reader.read(pathname=filename, model_name=_parse_problem_name(filename))
  File "/home/ima/envs/qiskit/lib/python3.8/site-packages/docplex/mp/model_reader.py", line 167, in read
    return mri.read_model(pathname, model_name, verbose, model_class, **kwargs)
  File "/home/ima/envs/qiskit/lib/python3.8/site-packages/docplex/mp/model_reader.py", line 303, in read_model
    raise RuntimeError("ModelReader.read_model() requires CPLEX runtime.")
RuntimeError: ModelReader.read_model() requires CPLEX runtime.
```